bower.json

```diff
@@ -1,14 +1,14 @@
 {
   "author": "Mozilla",
   "name": "marketplace-core-modules",
-  "version": "2.1.0",
+  "version": "3.0.0",
   "ignore": [
     "bower.json",
     "LICENSE",
     "README.md"
   ],
   "dependencies": {
-    "jquery": "2.0.2",
+    "marketplace-jquery": "2.0.2",
```

Comment:
Yeah. Making it a Bower module isn't a big deal.

Plus, there may be cases later on where each different frontend projects want different builds of jQuery, where Bower's versioning can come in handy. It'll help as I look to trim jQuery down a bit more. https://bugzilla.mozilla.org/show_bug.cgi?id=990324